### PR TITLE
Page Visit intial

### LINKED ISSUE #5: Place Visit Page - initial skeleton
Create an initial Place Visit Page skeleton with stub links.

This should include html and js files specific for this page.

### PR DESCRIPTION
**Pull request by chuckluv**
#5

### CODE REVIEW

File: frontend/src/main/webapp/index.html
Comment (cli1903) [inline frontend/src/main/webapp/index.html]: I think you left an extra space here by accident

File: frontend/src/main/webapp/pagevisit.html
Comment (cli1903) [inline frontend/src/main/webapp/pagevisit.html]: I think you accidentally left another space here and also in the div with the image

File: frontend/src/main/webapp/pagevisit.js
Comment (cli1903) [inline frontend/src/main/webapp/pagevisit.js]: was there a reason why you capitalized this function?

File: frontend/src/main/webapp/pagevisit.html
Comment (cli1903) [inline frontend/src/main/webapp/pagevisit.html]: can you add a space between the ul and { here?

File: frontend/src/main/webapp/pagevisit.js
Comment (chuckluv) [reply frontend/src/main/webapp/pagevisit.js]: No, I just forget to change it

**Comment by chuckluv** [comment]
@googlebot I signed it

**Comment by chuckluv** [comment]
@googlebot I signed it!

File: frontend/src/main/webapp/pagevisit.html
Comment (cli1903) [inline frontend/src/main/webapp/pagevisit.html]: there is an extra space here

File: frontend/src/main/webapp/pagevisit.html
Comment (cli1903) [inline frontend/src/main/webapp/pagevisit.html]: nit: add newline to end of file

File: frontend/src/main/webapp/pagevisit.js
Comment (cli1903) [inline frontend/src/main/webapp/pagevisit.js]: nit: add newline to end of file

**Review by cli1903 — DISMISSED** [review]

File: frontend/src/main/webapp/pagevisit.html
Comment (anfee1) [inline frontend/src/main/webapp/pagevisit.html]: I'm not absolutely sure, but I don't think we can have in line style or java.

Comment (chuckluv) [reply frontend/src/main/webapp/pagevisit.html]: yeah I didn't plan on keeping it it just I didn't have Css file

**Review by KoopaKing — DISMISSED** [review]
please address other comments before submitting.

File: frontend/src/main/webapp/pagevisit.html
Comment (anfee1) [reply frontend/src/main/webapp/pagevisit.html]: okay.

File: frontend/src/main/webapp/pagevisit.html
Comment (cli1903) [inline frontend/src/main/webapp/pagevisit.html]: one last space before the =

**Review by cli1903 — DISMISSED** [review]

**Review by cli1903 — DISMISSED** [review]

File: frontend/src/main/webapp/pagevisit.js
Comment (bsidhom) [inline frontend/src/main/webapp/pagevisit.js]: Same comments apply as in the trip list page (#20). Specifically, this function could use a clearer name and it would be better if you could use a simulated `fetchVisit` call.

**Review by bsidhom — DISMISSED** [review]
LGTM after a small rename and refactor based on comments in #20.

**Review by anfee1 — APPROVED** [review]
LGTM

File: frontend/src/main/webapp/index.html
Comment (KoopaKing) [inline frontend/src/main/webapp/index.html:10]: not sure why this is here...

**Review by KoopaKing — APPROVED** [review]

File: frontend/src/main/webapp/index.html
Comment (chuckluv) [reply frontend/src/main/webapp/index.html:10]: After I fixed the merge conflict I decided just to make both of the index.html I was working on the same

**Pull request merged**
